### PR TITLE
Initial update on PR page

### PR DESCRIPTION
**Pull request by MylesJarvis**
### Context
We're getting a new PR report soon, this PR updates the page to have previous content from the Legacy site.

### Changes proposed in this pull request

- [x] Add Text
- [x] Updated download links

### Guidance to review
Do the download links work?

For reference. Wayback machine link: https://web.archive.org/web/20210226191425/getintoteaching.education.gov.uk/research

### CODE REVIEW

File: app/views/content/research.md [hunk 1]
```diff
@@ -19,3 +19,96 @@ The purpose of the survey was aimed at gaining further insights about those who
...
+    Read the summary of findings (PDF 424KB)
+    <i class="fas fa-file-pdf"></i>
+</a>
+
+Published 29 November 2017
+Last updated 6 April 2018
+
+## Soft messaging survey
+
+On behalf of NCTL, SKV Communications and Kindred Agency commissioned Censuswide to carry out a survey. This asked teachers and the non-teaching public from other professions about the extent to which they experience emotionally rewarding benefits from their career.
```
Comment (peteryates) [inline app/views/content/research.md:107]: Worth explaining who NCTL are or maybe linking to their homepage?

File: app/views/content/research.md [hunk 2]
```diff
@@ -19,3 +19,96 @@ The purpose of the survey was aimed at gaining further insights about those who
...
+<a href="/assets/documents/it-takes-you-to-teach.pdf" class="call-to-action-icon-button">
+    Read the summary of findings (PDF 764KB)
+    <i class="fas fa-file-pdf"></i>
+</a>
+
+Published 23 September 2020
+
+## Exploring parents’ appreciation for teachers and experience of supporting home learning during COVID-19
+
+Kindred Agency, in partnership with SKV Communications, commissioned research company Censuswide to carry out a survey on behalf of the Get Into Teaching campaign, asking parents of primary and secondary school-aged children about how they feel towards teachers and their experience of home learning, during the coronavirus outbreak when schools remained closed to most pupils.
```
Comment (peteryates) [inline app/views/content/research.md:38]: Perhaps link to SKV?

**Review by peteryates — COMMENTED** [review]
Looks good overall, think it might be worth linking out to the agencies/companies (or at least expanding the acronyms) though

File: app/views/content/research.md (hunk 1 above)
Comment (MylesJarvis) [reply app/views/content/research.md]: Done

**Review by peteryates — DISMISSED** [review]

**Review by peteryates — APPROVED** [review]

**Pull request merged**